ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -730,6 +1229,181 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-gcp-upi
+- as: gcp-upi-private-xpn-ingress-glb-arm-f28
+  cron: 15 17 4 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      BOOTSTRAP_NODE_TYPE: t2a-standard-4
+      COMPUTE_NODE_TYPE: t2a-standard-4
+      CONTROL_PLANE_NODE_TYPE: t2a-standard-4
+      OCP_ARCH: arm64
+    test:
+    - chain: openshift-upgrade-qe-test
+    workflow: cucushift-installer-rehearse-gcp-upi-private-xpn-ingress-glb
+- as: ibmcloud-ipi-mini-perm-f28
```

Comment:
Again, ibmcloud/nutanix/vsphere is never officially support multi arch payload. I already did a correction last time in https://github.com/openshift/release/pull/54733

Reply:
Wierd, maybe I was using an out dated e2e config file. I will remove all those jobs.